js/misc/paygManager.js

```diff
@@ -60,6 +63,27 @@ const DBusErrorsMapping = {
     DISABLED          : 'com.endlessm.Payg1.Error.Disabled',
 };
 
+// Title and description text to be shown in the periodic reminders.
+const NOTIFICATION_TITLE_FORMAT_STRING = _("Pay-as-you-go time will expire in %s");
```

Comment:
I did not notice this before, and I'm not sure if "Pay-as-you-go" is intended to be a user-facing name of the feature.  That said, I'll merge as is for now, but there is a chance that we may have to tweak that string.

Reply:
I took that from Robin, and it's also consistent with the designs in invision (see T21811), but I also share your concern.